src/test/java/org/terasology/blockdetector/systems/BlockDetectorSystemTest.java

```diff
@@ -24,40 +10,41 @@
 import org.terasology.blockdetector.utilities.LinearAudioDetectorImpl;
 import org.terasology.engine.audio.AudioManager;
 import org.terasology.engine.context.Context;
+import org.terasology.engine.integrationenvironment.ModuleTestingHelper;
+import org.terasology.engine.integrationenvironment.jupiter.Dependencies;
+import org.terasology.engine.integrationenvironment.jupiter.MTEExtension;
+import org.terasology.engine.integrationenvironment.jupiter.UseWorldGenerator;
 import org.terasology.engine.logic.players.LocalPlayer;
 import org.terasology.engine.logic.players.event.ResetCameraEvent;
 import org.terasology.engine.registry.In;
 import org.terasology.engine.world.WorldProvider;
 import org.terasology.engine.world.block.BlockManager;
 import org.terasology.engine.world.block.BlockRegion;
-import org.terasology.moduletestingenvironment.MTEExtension;
-import org.terasology.moduletestingenvironment.ModuleTestingHelper;
-import org.terasology.moduletestingenvironment.extension.Dependencies;
-import org.terasology.moduletestingenvironment.extension.UseWorldGenerator;
 
 import java.io.IOException;
+import java.util.Set;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @ExtendWith(MTEExtension.class)
-@UseWorldGenerator("ModuleTestingEnvironment:empty")
-@Dependencies({"BlockDetector"})
+@UseWorldGenerator("unittest:dummy")
```

Comment:
Why do we use `unittest:empty` here and in the other module changes we use `unittest:dummy`?
I guess it doesn't matter too much, but would be nice to do it consistently anyway?

Reply:
nvm, looks like these two are different ones and `dummy` seems to be the correct choice here which also explains the difference in the test failures.